lib/zendesk_apps_tools/command.rb

```diff
@@ -52,8 +55,8 @@ def new
 
       prompt_new_app_dir
 
-      skeleton = options[:v1] ? 'app_template' : 'app_template_iframe'
-      is_custom_iframe = !options[:v1] && @iframe_location != 'assets/iframe.html'
+      skeleton = 'app_template_iframe'
```

Comment:
since this is only used in 1 spot maybe remove it and replace `skeleton` on line 61 with `'app_template_iframe'`?